src/sorcha/modules/PPCalculateApparentMagnitudeInFilter.py

```diff
@@ -46,14 +53,14 @@ def PPCalculateApparentMagnitudeInFilter(
 
     H_col = "H_filter"
 
-    # first, get H, r, delta and alpha as ndarrays
-    # r, delta and alpha are converted to au from kilometres
-    r = padain["AstRange(km)"].values / 1.495978707e8
+    # first, get H, rho, delta and alpha as ndarrays
+    # delta, rho and alpha are converted to au from kilometres
+    delta = padain["AstRange(km)"].values / 1.495978707e8
 
     try:
-        delta = padain["Ast-Sun(km)"] / 1.495978707e8
+        rho = padain["Ast-Sun(km)"] / 1.495978707e8
     except KeyError:
-        delta = (
+        rho = (
             np.sqrt(
                 padain["Ast-Sun(J2000x)(km)"].values ** 2
                 + padain["Ast-Sun(J2000y)(km)"].values ** 2
```

Comment:
Same thing. Can we get rid of the hardcoded numbers to handle #485

Reply:
Done.